rh-tictactoe/src/services/userService.js

```diff
@@ -10,4 +10,11 @@ const setInfo = (email, token) => {
   api.setHeader('Authenticated', token);
 };
 
-export { UserService, setInfo };
+const cleanInfo = () => {
+  localStorage.clear();
```

Comment:
mmmm you can be deleting info you dont want to delete from another application. Beware of this